client/dashboard/domains/domain-overview/featured-card-privacy.tsx

```diff
@@ -0,0 +1,20 @@
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { unseen } from '@wordpress/icons';
+import { Domain } from '../../data/domain';
+import OverviewCard from '../../sites/overview-card';
+
+interface Props {
+	domain: Domain;
+}
+
+export default function FeaturedCardPrivacy( { domain }: Props ) {
+	return (
+		<OverviewCard
+			title={ __( 'Privacy' ) }
+			heading={ __( 'WHOIS Privacy' ) }
+			icon={ <Icon icon={ unseen } /> }
+			description={ domain.privacy_available ? __( 'Enabled' ) : __( 'Disabled' ) }
```

Comment:
The `privacy_available` property indicates whether the TLD supports privacy protection. The `private_domain` property indicates whether privacy protection is currently enabled.

If the TLD doesn't support privacy, we should probably display a message like:
`Privacy protection is not available due to the registry’s policies.` (as in the current implementation.)

![Screenshot 2025-08-28 alle 13 35 30](https://github.com/user-attachments/assets/52940e49-f8c4-4cc6-9611-d97bc2f270b4)